test/aws-xray-sdk/tc_context.rb

```diff
@@ -20,14 +20,15 @@ def test_change_context_missing
     context = XRay::DefaultContext.new
     context.clear!
     context.context_missing = 'UNKWON'
```

Comment:
nit: is this a typo?

Reply:
It is, but I cannot tell if it is deliberate. The purpose of this line is to set a random invalid context_missing value, so this typo could either be on purpose or not. Setting it to 'UNKNOWN' will yield the same result.

Reply:
Makes sense. If they both yield the same result, maybe it would be better to just correct the typo for readability? What do you think?

Reply:
Changed to 'INVALID_STRATEGY'